packages/currency/test/currencyManager.test.ts

```diff
@@ -709,4 +709,46 @@ describe('CurrencyManager', () => {
       expect(path).toMatchObject([eur.hash, dai.hash.toLowerCase()]);
     });
   });
+
+  describe('Native and bridged USDC', () => {
```

Comment:
The `describe` and `it` should give details on the behaviour you are testing, not about the data.